### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by anmolbabu**
Signed-off-by: anmolbabu <anmolbudugutta@gmail.com>

### CODE REVIEW

File: tendrl/performance_monitoring/central_store/__init__.py [hunk 1]
```diff
@@ -163,6 +163,12 @@ def get_cluster_alerts(self, cluster_id):
 
     def get_cluster_summary(self, cluster_id):
         try:
+            # Blank read as object#load returns empty object instead of error
+            # if obj doesn't actually exist in etcd. Now, if object is not
+            # available in etcd it would fail EtcdKeyNotFound and be handled.
+            NS.etcd_orm.client.read(
```
Comment (r0h4n) [inline tendrl/performance_monitoring/central_store/__init__.py]: Use "exists()" 

https://github.com/Tendrl/commons/blob/develop/tendrl/commons/objects/__init__.py#L120

File: tendrl/performance_monitoring/central_store/__init__.py [hunk 2]
```diff
@@ -176,10 +182,16 @@ def get_cluster_summary(self, cluster_id):
...
         except Exception as ex:
-            TendrlPerformanceMonitoringException(str(ex))
+            raise TendrlPerformanceMonitoringException(str(ex))
 
     def get_system_summary(self, cluster_type):
         try:
+            # Blank read as object#load returns empty object instead of error
+            # if obj doesn't actually exist in etcd. Now, if object is not
+            # available in etcd it would fail EtcdKeyNotFound and be handled.
+            NS.etcd_orm.client.read(
```
Comment (r0h4n) [inline tendrl/performance_monitoring/central_store/__init__.py]: same as above comment 

https://github.com/Tendrl/commons/blob/develop/tendrl/commons/objects/__init__.py#L120

File: tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py [hunk 3]
```diff
@@ -96,6 +97,8 @@ def get_brick_status_wise_counts(self, volumes_det, cluster_id):
                 'Bricks',
                 {}
             ).iteritems():
+                if isinstance(brick_det, basestring):
```
Comment (r0h4n) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: no need for type checking, catch the exception thrown by json.loads

File: tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py [hunk 4]
```diff
@@ -173,6 +176,8 @@ def get_most_used_bricks(self, volumes_det, cluster_name):
...
```
Comment (r0h4n) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: no need for type checking, catch the exception thrown by json.loads

File: tendrl/performance_monitoring/central_store/__init__.py (hunk 1 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/central_store/__init__.py]: Done

File: tendrl/performance_monitoring/central_store/__init__.py (hunk 2 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/central_store/__init__.py]: Done

File: tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py (hunk 4 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: Done

File: tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py (hunk 3 above)
Comment (anmolbabu) [inline tendrl/performance_monitoring/sds/glusterfs/glusterfs_plugin.py]: Done

File: tendrl/performance_monitoring/central_store/__init__.py [hunk 5]
```diff
@@ -176,13 +181,21 @@ def get_cluster_summary(self, cluster_id):
                 del summary['list']
             return summary
         except Exception as ex:
-            TendrlPerformanceMonitoringException(str(ex))
+            raise TendrlPerformanceMonitoringException(str(ex))
 
     def get_system_summary(self, cluster_type):
         try:
+            # Blank read as object#load returns empty object instead of error
```
Comment (r0h4n) [inline tendrl/performance_monitoring/central_store/__init__.py]: remove this comment

Comment (anmolbabu) [inline tendrl/performance_monitoring/central_store/__init__.py]: Done

**Pull request merged**